Makefile

```diff
@@ -61,3 +61,20 @@ binaries: ${PREFIX}/bin/registry ${PREFIX}/bin/registry-api-descriptor-template
 clean:
 	@echo "+ $@"
 	@rm -rf "${PREFIX}/bin/registry" "${PREFIX}/bin/registry-api-descriptor-template"
+
+# import the existing docs build cmds from docker/docker
+DOCS_MOUNT := $(if $(DOCSDIR),-v $(CURDIR)/$(DOCSDIR):/$(DOCSDIR))
+DOCSPORT := 8000
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
```

Comment:
mmmm, true - this is taken from what we did in `docker/docker` - though `uniquely` isn't entirely true - its only unique to a particular clone - so at some point we're fuzzy-matching.